### PR TITLE
mraa & upm: Updated to latest version

### PR DESCRIPTION
**Pull request by YinThong**
mraa update version from 1.6.0 to 1.7.0
upm update version from 1.1.0 to 1.3.0

Signed-off-by: Choong YinThong <yin.thong.choong@intel.com>

### CODE REVIEW

File: meta-refkit-core/recipes-sensors/mraa/mraa_git.bb [hunk 1]
```diff
@@ -4,9 +4,12 @@ AUTHOR = "Brendan Le Foll, Tom Ingleby, Dmitry Rozhkov"
...
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://COPYING;md5=66493d54e65bfc12c7983ff2e884f37f"
+PV = "1.7.0"
 
-SRC_URI = "git://github.com/intel-iot-devkit/mraa.git;protocol=http;tag=v${PV} \
-           "
+SRC_URI = "git://github.com/intel-iot-devkit/${BPN}.git;protocol=http; \
+          "
+
+SRCREV = "${AUTOREV}"
```
Comment (mythi) [inline meta-refkit-core/recipes-sensors/mraa/mraa_git.bb]: There's an eSDK bug about AUTOREV usage: https://bugzilla.yoctoproject.org/show_bug.cgi?id=11350. It means we cannot use AUTOREV yet. Let's go with the more traditional way (```PV = "1.7.0-${SRCPV}"``` and ```SRCREV````)

**Comment by YinThong** [comment]
test this please

**Comment by mythi** [comment]
@YinThong the changes look OK overall but we don't need the intermediate commits. Please squash/fixup the changes in two (e.g., with ```git rebase -i HEAD~5```) and force push

File: meta-refkit-core/recipes-sensors/upm/upm_git.bb [hunk 2]
```diff
@@ -7,7 +7,11 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=66493d54e65bfc12c7983ff2e884f37f"
 
 DEPENDS = "libjpeg-turbo mraa"
 
-SRC_URI = "git://github.com/intel-iot-devkit/upm.git;protocol=http;tag=v${PV}"
+SRCREV = "106b6c706268522ab0168a4ddb19e89ce832e084"
+PV = "1.3.0-git${SRCPV}"
+
+SRC_URI = "git://github.com/intel-iot-devkit/${BPN}.git;protocol=git; \
```
Comment (mythi) [inline meta-refkit-core/recipes-sensors/upm/upm_git.bb]: let's use ```protocol=http```

let's also drop ```;``` after protocol setting for both mraa and upm recipes

**Pull request merged**